bastionzero/zli
# Fix/close ssh tunnels on logout [CWC-628]

## Description of the change

This fixes the bug when there are multiple instances of the zli running and one of them logs out, the others would not pick up the change and close any open tunnels.

The suggested fix is the following:
When a zli logs out, before exiting, it lists all the open cli connections and sends for each one of them a close connection request to the back end. The back end then closes them from its side and this way all the open tunnels are getting closed as well.

In order to test this:
- Open a terminal. Login to zli. Connect to a server from this terminal.
- Open a second terminal. Execute logout.
- Check both terminals. Both should be disconnected and logged out without any error messages. Make sure you look in the logs of both the zli and the webapp please, we have noticed some discrepancies and we want to make sure those do not show up anymore.
- Extra test : try logging in and connecting again to the server.

## Type of change


## Related JIRA tickets

Relates to JIRA: CWC-628

## Potential Security Impacts

## Checklists

### Development
